cmd/wsh/cmd/wshcmd-file.go

```
@@ -58,140 +88,111 @@ func init() {
 	fileCmd.AddCommand(fileRmCmd)
 	fileCmd.AddCommand(fileInfoCmd)
 	fileCmd.AddCommand(fileAppendCmd)
-	fileCmd.AddCommand(fileCpCmd)
+	// fileCpCmd.Flags().BoolP("merge", "m", false, "merge directories")
+	// fileCpCmd.Flags().BoolP("recursive", "r", false, "copy directories recursively")
+	// fileCpCmd.Flags().BoolP("force", "f", false, "force overwrite of existing files")
+	// fileCmd.AddCommand(fileCpCmd)
 }
 
 type waveFileRef struct {
```

golangci-lint (1.62.2)

type `waveFileRef` is unused

(unused)